tests/bluetooth/bsim_bt/bsim_test_l2cap_ecred/src/main.c

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "bstests.h"
+
+extern struct bst_test_list *test_main_l2cap_ecred_install(struct bst_test_list *tests);
```

Comment:
I would be in favor of splitting the central and peripheral into two files. It makes it a lot easier to determine what code is run on which side, and anything common can be put in a common file.

Reply:
I aggrege with that